vsintegration/src/FSharp.ProjectSystem.FSharp/AppConfigHelper.fs

```diff
@@ -362,7 +362,7 @@ namespace Microsoft.VisualStudio.FSharp.ProjectSystem
                         "4.3.0.0",    "4.4.3.0",    ["2.0.0.0";    "2.3.0.0";   "2.3.5.0";    "4.0.0.0";   "4.3.0.0"]
                         "4.3.1.0",    "4.4.3.0",    ["3.3.1.0";    "2.3.5.1";   "3.78.3.1";   "3.259.3.1"; "4.3.1.0"]
                         "4.4.0.0",    "4.4.3.0",    ["3.47.4.0";   "3.78.4.0";  "3.259.4.0";  "4.4.0.0"]
-                        "4.4.1.0",    "4.4.3.0",    ["3.47.41.0";  "3.78.41.0"; "3.259.41.0"; "4.4.0.0";   "4.4.3.0"]
+                        "4.4.1.0",    "4.4.3.0",    ["3.47.41.0";  "3.78.41.0"; "3.259.41.0"; "4.4.1.0";   "4.4.3.0"]
```

Comment:
4.4.0.0 should be in this list

Reply:
Should there be a separate line for ``4.4.3.0`` in this table?

Reply:
No to both ...

The correct binding redirects for 4.4.0.0 are produced by this line:
"4.4.0.0",    "4.4.3.0",    ["3.47.4.0";   "3.78.4.0";  "3.259.4.0";  "4.4.0.0"] 

I think the "4.4.1.0" is not needed and the orginal "4.4.0.0" was redundent.  I will check it out.